src/command-registry.js

```diff
@@ -397,10 +394,41 @@ class SelectorBasedListener {
       this.sequenceNumber - other.sequenceNumber
     )
   }
+
+  getPublicCommand () {
+    return getPublicCommand.call(this)
```

Comment:
`command` is the internal name of the structure inside the listener, and you call `getPublicCommand` to get a version that's safe to hand around externally (basically it doesn't have the `handleEvent` on it)